### PR TITLE
fix: Avoid delay when hovering over SelectorItem with mouse

### LINKED ISSUE #12016: Hovering delay should be removed when using mouse
### Current behavior

https://github.com/unoplatform/uno/blob/97d60a9cd1cfe8ebeb07172e804a5afbf8e13abc/src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs#L47-L50

We shouldn't have a delay when using mouse.

### Expected behavior

_No response_

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #12016

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by Youssef1313** [comment]
@dr1rrb Aside from WinUI build failures, does this look good? 👀

File: src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs [hunk 1]
```diff
@@ -170,6 +170,12 @@ private void UpdateCommonStates(ManipulationUpdateKind manipulationUpdate = Mani
 			{
 				// We delay the beginning of a manipulation to avoid flickers, but not for "exact" devices which has hover states
 				// (i.e. mouse and pen when not on iOS)
```
Comment (dr1rrb) [inline src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs:172]: About pen: I've confirmed on WinUI that we can scroll a `ListView` using the pen just like a finger This means that we want to apply the same delay on the over state to improve performance of the scrolling.

You should update the comment to reflect it :)

**Review by dr1rrb — COMMENTED** [review]
> @dr1rrb Aside from WinUI build failures, does this look good? 👀

Yes 👍

This is something that can be tested with runtime tests and injected pointers on skia

**Review by agneszitte — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs (hunk 1 above)
Comment (Youssef1313) [reply src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs:173]:
```suggestion
				// (i.e. mouse when not on iOS)
```

**Review by dr1rrb — APPROVED** [review]

**Pull request merged**